processing/src/test/java/org/apache/druid/query/aggregation/FilteredAggregatorTest.java

```diff
@@ -218,14 +253,22 @@ public ColumnCapabilities getColumnCapabilities(String columnName)
 
   private void assertValues(FilteredAggregator agg, TestFloatColumnSelector selector, double... expectedVals)
   {
-    Assert.assertEquals(NullHandling.defaultDoubleValue(), agg.get());
-    Assert.assertEquals(NullHandling.defaultDoubleValue(), agg.get());
-    Assert.assertEquals(NullHandling.defaultDoubleValue(), agg.get());
-    for (double expectedVal : expectedVals) {
```

Comment:
maybe this should accept `Double...` instead of `double...` so that it confirms that it _actually_ tests the expected values, since in this case the selectory always returns null it seems like for the test cases using `TestFloatNullableColumnSelector` this `!agg.isNull` branch will never be hit...